src/nvim/if_cscope.c

```diff
@@ -778,7 +778,6 @@ static int cs_create_connection(size_t i)
     if (execl("/bin/sh", "sh", "-c", cmd, (char *)NULL) == -1)
       PERROR(_("cs_create_connection exec failed"));
 
-    stream_set_blocking(input_global_fd(), true);  // normalize stream (#2598)
```

Comment:
Why could this be removed?

Reply:
Because it made no sense in the first place? We are in a subprocess with replaced fds, so `input_global_fd()` doesn't mean anything.